### PR TITLE
Logo at top of sidebar

### LINKED ISSUE #263: Logo at top of sidebar
maybe worth trying: https://arnaudmiribel.github.io/streamlit-extras/extras/app_logo/

### PR DESCRIPTION
**Pull request by joAschauer**
Fixes #263

### CODE REVIEW

File: app/sidebar.py [hunk 1]
```diff
@@ -7,7 +7,15 @@
...
 
+@st.cache_resource()
+def sidebar_logo():
+    st.image("img/agora-energiewende_logo_612x306.png")
+
+
 def make_sidebar(api: PtxboaAPI):
+    with st.sidebar:
+        sidebar_logo()
+
```
Comment (joAschauer) [inline app/sidebar.py:18]: I tried the new st.logo. You can uncomment this to see how it looks with texts removed from the logos.

File: app/sidebar.py [hunk 2]
```diff
@@ -7,7 +7,24 @@
...
+        image="img/logos_wide_text_1000x100.png",  # gets scaled to 240x24 by streamlit
+        icon_image="img/agora-energiewende_logo_612x306.png",
+        link="https://agora-energiewende.org",
+    )
+
+    # st.logo(  # noqa E800
+    #     image="img/logos_wide_no_text_1000x100.png",  # noqa E800
+    #     icon_image="img/logo_agora_no_text_257x100.png",  # noqa E800
+    #     link="https://agora-energiewende.org",  # noqa E800
+    # )  # noqa E800
```
Comment (joAschauer) [inline app/sidebar.py:26]: I tried the new st.logo. You can uncomment this to get logos with texts removed. The logos don't get larger with st.logo. A larger logo was there with f48b477fd340928d11869ca8149cd880d0e294a1

Comment (markushal) [reply app/sidebar.py:26]: That is too small imo. I would prefer to use just one of the four logos, but with text. Lets have Agora decide which logo they want. And if it still is too small, we can just include an image without using st.logo, right?

**Review by markushal — CHANGES_REQUESTED** [review]
- use only agora energiewende logo, but with text
- resolve conflicts with https://github.com/agoenergy/ptx-boa/pull/393

**Review by markushal — APPROVED** [review]
Thanks. I guess that is acceptable :)

**Pull request merged**